src/test/scala/mesosphere/marathon/core/health/impl/HealthCheckWorkerActorTest.scala

```diff
@@ -87,56 +83,5 @@ class HealthCheckWorkerActorTest extends AkkaUnitTest with ImplicitSender {
       watch(ref)
       expectTerminated(ref)
     }
-
```

Comment:
This test has been moved to the [MarathonHttpHealthCheckActorTest.scala](https://github.com/mesosphere/marathon/pull/6840/files#diff-d032c01757343484c47409495456c6f4) below